tests/tests/serve.rs

```diff
@@ -477,3 +477,36 @@ fn ref_properties_remove() {
         );
     });
 }
+
+#[test]
+fn ref_properties_patch_update() {
```

Comment:
Maybe add some comments to this test to describe what we expect to happen here, and why this test was added? Maybe link to the issue